fdasrsf/utility_functions.py

```diff
@@ -131,6 +131,7 @@ def optimum_reparam(q1, time, q2, method="DP", lam=0.0, f1o=0.0, f2o=0.0):
     :param q2: vector of size N or array of NxM samples samples of second SRSF
     :param method: method to apply optimization (default="DP") options are "DP" and "DP2"
```

Comment:
Lets make the default DP2 now

Reply:
Good question, they are both the same in the sense of the algorithm, however DP2 can handle different sample spacing for q1 and q2. DP is the original code from the work and kept for comparison. Anuj prefers it as it was the original and I have kept it the default for him.

Reply:
But the results were very different with the two methods, at least in some of my tests.

Reply:
So I should qualify, they are both implementing Dynamic Programming, they are not the same in implementation. I wrote the second version and have not wrote DP. The differences come from implementation of the spline.